nilearn/decomposition/tests/test_canica.py

```diff
@@ -3,17 +3,34 @@
 import numpy as np
 from numpy.testing import assert_array_almost_equal
 from nose.tools import assert_true, assert_raises
-
 import nibabel
-
 from nilearn.decomposition.canica import CanICA
 
 
-def test_canica_square_img():
-    shape = (20, 20, 1)
-    affine = np.eye(4)
-    rng = np.random.RandomState(0)
+try:
```

Comment:
Could you please move this to a new file in nilearn/_utils/fixes/testings and mention in a comment that when we support scikit-learn 0.14 we can remove it.